### ISSUE TITLE
arangorestore: add option --data-only default to false

### ISSUE THREAD

**Issue by sleto-it** [arangodb/arangodb#3732]
Affected version: 3.2 + devel

It would be useful to have a way to restore from a dump only the data (without restoring the collection structures)

This is the use case:

- take a backup using dump
- manually create a new database where to restore, as well as the collections structures (manually or using a script)
- use arangorestore to restore just the data (\*.data.json), ignoring the structures (\*.structure.json)

It may be done by specifying collections, e.g.

```
arangorestore  --data-only --collection coll1--collection coll2--input-directory "dump"
```

or directly for all collections that are found in the dump directory:

```
arangorestore  --data-only --input-directory "dump"
```

If a collection does not exist in the target database, then a warning message may be written and execution continue with the next collection

**Complete use case:**

```
db._createDatabase("original");
db._useDatabase("original");
db._create("coll1");
db._create("coll2");

[insert data into coll1 and coll2]

arangodump --server.endpoint tcp://localhost:8529 --server.username root --server.database original --output-directory "dump"
```

On the same server:

```
db._createDatabase("restore");
db._useDatabase("restore");
db._create("coll1");
db._create("coll2");

arangorestore  --data-only --collection coll1--collection coll2--input-directory "dump"
```

This would workaround Issue #2437, and related ones

**Comment by jsteemann** [comment]
It is possible to use the existing option `--create-collection` and set its value to `false`.
This will do exactly what is required. It turned out the arangorestore documentation was slightly off regarding this option. The PRs above address this.

**Comment by jsteemann** [comment]
Documentation changes are merged into 3.2 and devel.

**Comment by sleto-it** [comment]
Command would be:

arangorestore --server.username username --server.database restore  --create-collection = false --input-directory "dump"
